### DISCUSSION TITLE
Where is the "Pull Request" video?

### DISCUSSION THREAD

**Discussion by reneoliveirajr** [skills/.github#218] [Q&A]
In "Introduction to Github", step 3 we see "or watch the video linked below.". But I could not find the video.

It is an issue? Or a missed something?

Best Regards!! (This is my first discussion in Github Community!! :)

**Comment by sinsukehlab** [comment]
They have removed the broken video links for now.
See: issue skills/introduction-to-github#346 PRs skills/introduction-to-github#356 skills/introduction-to-github#357

**Comment by cmwilson21** [comment]
@reneoliveirajr Thanks for reporting this! Looks like we missed one of the references to the video, so I went ahead and removed it from step 3. Thanks again!

**Reply by sinsukehlab** [reply]
I'm eagerly awaiting the newer version of the deleted videos.
